src/quo2/components/base_tag.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo2.components.base-tag
+  (:require
+   [quo.react-native :as rn]))
+
+(defn style-container [size disabled border-color border-width background-color label type]
+  (merge {:height             size
+          :border-color       border-color
+          :background-color   background-color
+          :border-width       border-width
+          :border-radius      size
+          :align-items        :center
+          :justify-content    :center}
+         (when disabled
+           {:opacity 0.3})
+         (when (and (or (= type :icon) (= type :emoji)) (not label))
+           {:width       size})))
+
+(defn base-tag
+  "opts
+   {:type :icon/:emoji/:label/:permission
+    :size 32/24}
+    :labelled true"
+  [_]
+  (fn [{:keys [id size disabled border-color border-width background-color on-press
+               accessibility-label label type] :or   {size 32}} children]
```

Comment:
Thank you very much for adding comment, But it don't look like it matches parameters.